### ISSUE TITLE
Bulkloader Fails to Build

### ISSUE THREAD

**Issue by cjduffett** [synthetichealth/bulkfhirloader#10]
Just cloned the latest bulkloader from master and it fails to build. This is directly related to the last pull request that added age range support. In the future please build and run the binary locally before submitting a PR. The following issue stops the build:

```
./main.go:286: not enough arguments in call to bulkloader.UploadResources
```

**Comment by gregquinn2001** [comment]
Whoever reviews and merges a pull request should also test it to make sure it all works outside the creators dev environment. A pull request shouldn't be approved unless it works in other words.

**Comment by cjduffett** [comment]
@gregquinn2001, agreed - a lack of oversight on my part. After playing around locally and fixing the build I also discovered that the bulkloader is not compatible with the latest synthea output. Synthea is now on FHIR 1.8 and the bulkloader, gofhir server, and UI are still on FHIR 1.6. We'll need to coordinate updates to all of these tools.

**Comment by gregquinn2001** [comment]
Yes, we should log a separate github issue against SM for updating to FHIR 1.8.
